filebeat/docs/inputs/input-kafka.asciidoc

```diff
@@ -27,6 +27,29 @@ Example configuration:
 
 ----
 
+[id="{beatname_lc}-input-{type}-azure"]
+==== Configuring for Azure Event Hubs
+
+The `kafka` input can be used to ingest data from Microsoft Azure Event Hubs
```

Comment:
Make this more active by saying something like: 

The following example shows how to use the `kafka` input to ingest data from Microsoft Azure Event Hubs that have Kafka compatibility enabled: